components/TxForm.tsx

```diff
@@ -1,34 +1,143 @@
 import useNotifications from "@/hooks/useNotifications";
+import useShieldPrivateKey from "@/hooks/useShieldPrivateKey";
 import { Button } from "@chakra-ui/button";
 import { FormControl, FormLabel } from "@chakra-ui/form-control";
 import { Input, InputGroup, InputRightElement } from "@chakra-ui/input";
 import { Box, Flex } from "@chakra-ui/layout";
 import { Select } from "@chakra-ui/select";
+import {
```

Comment:
_(arbitrary comment location to start a thread)_

When I click the WETH token (connect to goerli) it never actually selects as seen in this gif

![CleanShot 2023-01-12 at 13 47 17](https://user-images.githubusercontent.com/17163988/212188248-a5f07e8d-08e0-46d4-8ad2-3b308adeb8b2.gif)

Reply:
made a small fix so that this is no longer happening
we will be replacing this select component in #12